app/controller.py

```diff
@@ -164,8 +167,11 @@ def __init__(
             "E_nt": None,
             "I": None,
             "I_asym": None,
+            "I_asym0_rel": None,
```

Comment:
Hiermit müssen wir vermutlich aufpassen, es ist vermutlich nicht so gut (Konsistenz der Daten) zb I_asym an zwei Stellen zu speichern. Wäre aber Teil der insgesamten Problematik mit relativen Werten, also kann das ja theoretisch in zukunft mit #57 besser gelöst werden